go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -822,29 +824,6 @@ var HistorySystemTableScriptTests = []queries.ScriptTest{
 					{3, 4},
 				},
 			},
-			{
```

Comment:
it might be easier to leave these here with `SkipPrepared=true`, but still append to a new list of tests titled `BrokenHistorySystemTableScriptTests `.

Reply:
in which case you don't need the comment above